.travis.yml

```diff
@@ -1,24 +1,79 @@
-dist: xenial
 language: python
-python:
-- 3.7
-- 3.6
-- 3.5
-- 2.7
-install: pip install -U tox-travis coverage coveralls
+# install python on windows
+
+env:
+  global:
+    # Note: TWINE_PASSWORD is set in Travis settings
+    - TWINE_USERNAME=stevenjeanette
+    # skip py27 (EOL) and linux-32bit (Test get_h->test_nu0 hangs)
+    - CIBW_SKIP="cp27-* *-manylinux_i686"
+    # first versions to support py3.8
+    - CIBW_BEFORE_BUILD="pip install numpy==1.17.3 scipy==1.3.2 mpmath==1.1.0 setuptools"
```

Comment:
Can we not rely on the requirements in setup.py?

Reply:
I could set different versions for py35, py36, py37 and py38. The above versions are all the first ones to support py38.

Reply:
So if we have eg. `numpy>=1.16.x` or something in the `setup.py`, wouldn't it choose the newest version possible on the current python version? That also means we won't have to keep changing the dependency in this script every time numpy updates.

Reply:
I can remove that line, so we rely on the ``setup.py``. Then it will always use the latest version.

Reply:
Done